### ISSUE TITLE
32 bit

### ISSUE THREAD

**Issue by n9ami** [jontio/JAERO#82]
Dowloaded the latest however its only installing 64 bit. I did find another old 32 version  however I get no planes coming on the log even though the program is decoding things. Is there a work around for a newer 32 bit version?

**Comment by jontio** [comment]
Yup I haven't been building any 32bit versions for awhile now. Actually it's github that builds things these days. I'm not planning on writing something to build 32bit version. So you will have to build it yourself.
